docs/src/get-started/add-data/merge.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Git merge
+weight: -9
+description: Add a service and see how data works between branches.
+---
+
+You have a separate environment with separate data.
+Next, add a service to your development environment.
+
+## Add a service
+
+Platform.sh includes many services such as databases, cache, and search engines.
+These are included in your project, so you can manage them with Git and back them up with your project.
+
+Add a database service by following these steps:
+
+1. Create a services configuration file.
+
+   ```bash
+   touch .platform/services.yaml
+   ```
+
+   This file holds the configuration for all services your app needs.
+
+2. Add a database in that file.
```

Comment:
Not to overly complicate things, but we do have service snippets that are being pulled into console that could build the `services.yaml` file here....